scripts/generate_visualc_files.pl

```diff
@@ -49,9 +49,20 @@
 my $test_drivers_header_dir = 'framework/tests/include/test/drivers';
 my $test_drivers_source_dir = 'framework/tests/src/drivers';
 
-my @thirdparty_header_dirs = qw(
-    tf-psa-crypto/drivers/everest/include/everest
-);
+# This is a dirty patch to allow mbedtls#10091 to be merged without updating
+# tf-psa-crypto to psa#235. Once psa#235 will be merged, this dirty fix can
+# be removed.
+# The same holds also for @include_directories below.
+my @thirdparty_header_dirs;
```

Comment:
There's a lot of unnecessary duplication here. The code could be simpler:

```
my $everest_include_dir = (-d ... ? ... : ...);
...
my @include_directories = qw(...); # skipping everest dirs
push @include_directories, map {"$everest_include_dir/$_"} "", qw(everest ...);
```

Not a blocker because I think we should get rid of this script soon (but it's still undecided).